tutorials/getting-started-with-flow/02__listing-entities-in-a-grid/content.adoc

```diff
@@ -1,78 +1,94 @@
 [[flow.tutorial.grid]]
-== Listing Entities in a Grid
+= Listing entities in a Grid
```

Comment:
This is not English style title where nouns are with upper case. These where just changed to this by our book author, so I'd keep them as such.

Reply:
It was agreed in one of the Friday meetings to keep title case for all headings. The reason is that this is what is prevalent in the Vaadin documentation, and consistency is important.

Reply:
This was fixed already. Thanks.